packages/coq/coq.8.7.1/opam

```diff
@@ -0,0 +1,57 @@
+opam-version: "1.2"
+maintainer: "Ralf Jung <post@ralfj.de>"
+authors: "The Coq development team, INRIA, CNRS, University Paris Sud, University Paris 7, Ecole Polytechnique."
+homepage: "https://coq.inria.fr/"
+bug-reports: "https://github.com/coq/coq/issues"
+dev-repo: "https://github.com/coq/coq.git"
+license: "LGPL-2.1"
+build: [
+  [
+    "./configure"
+    "-configdir" "%{lib}%/coq/config"
+    "-prefix" prefix
+    "-mandir" man
+    "-docdir" doc
+    "-libdir" "%{lib}%/coq"
+    "-datadir" "%{share}%/coq"
+    "-usecamlp5"
+    "-camlp5dir" "%{lib}%/camlp5"
+    "-coqide" "no"
+  ]
+  [make "-j%{jobs}%"]
+  [make "-j%{jobs}%" "byte"]
+]
+install: [
+  [make "install"]
+  [make "install-byte"]
+]
+remove: [
+  ["rm" "-R" "%{lib}%/coq" "%{share}%/coq"]
+  ["rm"
+  "%{man}%/man1/coqc.1"
+  "%{man}%/man1/coqchk.1"
+  "%{man}%/man1/coqdep.1"
+  "%{man}%/man1/coqdoc.1"
+  "%{man}%/man1/coqide.1"
+  "%{man}%/man1/coq_makefile.1"
+  "%{man}%/man1/coqmktop.1"
+  "%{man}%/man1/coq-tex.1"
+  "%{man}%/man1/coqtop.1"
+  "%{man}%/man1/coqtop.byte.1"
+  "%{man}%/man1/coqtop.opt.1"
+  "%{man}%/man1/coqwc.1"
+  "%{man}%/man1/gallina.1"
+  "%{share}%/texmf/tex/latex/misc/coqdoc.sty"
+  "%{share}%/emacs/site-lisp/coq-font-lock.el"
+  "%{share}%/emacs/site-lisp/coq-inferior.el"
+  "%{share}%/emacs/site-lisp/gallina-db.el"
+  "%{share}%/emacs/site-lisp/gallina.el"
+  "%{share}%/emacs/site-lisp/gallina-syntax.el"
+  ]
+]
+depends: [
+  "ocamlfind"
```

Comment:
Could you add the ```build``` attribute to ```ocamlfind``` ? It seems that only the ```ocamlfind``` tool is used and not the library.
Is it also the case for ```campl5``` ?

Reply:
ocamlfind/camlp5 is necessary for those people wanting to develop against Coq. I am not sure what is the OPAM policy on `-dev` libraries [in the Debian sense], but it is required for Coq that people doing `opam install coq` are able to build Coq plugins afterwards.

Reply:
The ```build``` attribute just tells the OPAM solver to not rebuild the package in the dependency in question is upgraded. Nothing is changed for users or plugin developers.

Reply:
I see thanks, so then the answer is that the attribute should be added to ocamlfind, but not to `camlp5`.

Reply:
I did some googling but was unable to find information about "attributes" for opam dependencies. The word "attribute" also does not appear at <https://opam.ocaml.org/doc/Packaging.html>. Could you give me a pointer how I would add such an attribute?

Reply:
In this doc it appears in the "Build depends" section.
In the dev manual they are referred as "dependency flags" on page 7: https://github.com/ocaml/opam/blob/master/doc/dev-manual/dev-manual.pdf

Reply:
I see, thanks :)